code/modules/clothing/masks/hailer.dm

```diff
@@ -3,7 +3,7 @@
 
 /obj/item/clothing/mask/gas/sechailer
 	name = "security gas mask"
-	desc = "A standard issue Security gas mask with integrated 'Compli-o-nator 3000' device. Plays over a dozen pre-recorded compliance phrases designed to get scumbags to stand still whilst you taze them. Do not tamper with the device."
+	desc = "A standard issue Security gas mask with integrated 'Compli-o-nator 3000' device. Plays over a dozen pre-recorded compliance phrases designed to get scumbags to stand still whilst you tase them. Do not tamper with the device."
```

Comment:
Reverse triggered?

Reply:
This isn't actually a GB vs US thing, Taser is a brand, ergo the correct term would be tase/tased/tasing.

Reply:
Better remove tasers then before we get sued. Disablers all the way!